masonry/src/widgets/flex.rs

```diff
@@ -28,6 +28,7 @@ pub struct Flex {
     fill_major_axis: bool,
     children: Vec<Child>,
     old_bc: BoxConstraints,
+    child_removed: bool, // TODO: why can't we use WidgetState.children_changed instead?
```

Comment:
The reason we can't use `children_changed` here is two reasons:
1) It will always be handled "before" we get to layout. Indeed, that's the reason that it doesn't have a getter - at this point in the timeline, it doesn't have a meaningful value.
2) It doesn't handle spacers being removed

Reply:
That is, we should remove this comment.